src/scheduling/scheduling.service.ts

```diff
@@ -42,38 +41,40 @@ export class TasksService {
     ]);
 
     await Promise.all(
-      guilds.map(async (guild_ent) => {
-        // TODO: close all open events and start new ones in-place
+      guilds
+        .filter((guild_ent) => guild_ent.score.reset_enabled)
+        .map(async (guild_ent) => {
+          // TODO: close all open events and start new ones in-place
 
-        // Getting the Biggest Virgin for the announcement
-        const top_virgins = await this.virgins.find(
-          { guild: guild_ent.id },
-          { orderBy: [{ cached_dur_in_vc: -1 }], limit: 10 },
-        );
-        // send leaderboard to guild's vbot channel
-        const [leaderboard, channel] = await Promise.all([
-          this.leaderboard.buildLeaderboardEmbed(guild_ent),
-          this.discord_helper.findOrCreateVirginityBotChannel(guild_ent),
-        ]);
-        // TODO(4): "week" here assumes that our `score.reset_schedule` remains weekly
-        leaderboard.setTitle(`Last week's biggest virgins:`);
-        leaderboard.addFields({
-          name: ' ',
-          value: 'Scores have now been reset 😇',
-        });
+          // Getting the Biggest Virgin for the announcement
```

Comment:
was this all just a prettier formatting adjustment?

Reply:
I added a `filter` step up on [line 45](https://github.com/EdgarSaldivar/VirginityBot/pull/71/files#diff-f73723aaa9ba3e48d4dffdb3400a9a2fdc594b0b5070b6118625eaec8e525fedR45)

Reply:
ahhh kk missed that